core/templates/pages/welcome-guide-page/welcome-guide-page.directive.html

```diff
@@ -0,0 +1,67 @@
+<div style="height: auto; background-color: #FFFFFF;" class="row content">
+  <div  class="col-lg-7">
+    <div class="oppia-welcome-guide-header">
+      <h1 class="oppia-welcome-guide-h1" translate="I18N_WELCOME_GUIDE_TITLE"> </h1>
+      <p class = "oppia-welcome-guide-p" translate="I18N_WELCOME_GUIDE_PARAGRAH_1">
+      </p>
+
+    </div>
+  </div>
+  <div class="oppia-welcome-guide-img col-lg-5">
+    <img ng-src="<[$ctrl.headerImgUrl]>" alt="Learning with Oppia">
```

Comment:
Could you please replace these with modern version of compressed images (WebP format) added? You could modify it according to #8499. Ditto for other images in this HTML.

Reply:
fixed!